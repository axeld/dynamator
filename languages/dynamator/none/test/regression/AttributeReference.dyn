<dynamator language="none">
  <tag tag="body">
    <attr name="bgcolor">
      <content>[[@/f/d/
        +/d/c/
        +/d/e/
        ]]</content>
    </attr>
  </tag>
  
  <tag tag="a" with-attr="href">
    <attr name="href">
      <content>[[@/.html/.htm]]</content>
    </attr>
  </tag>

  <tag tag="img">
    <attr name="alt">
      <content>This image shows: [[@]]</content>
    </attr>
  </tag>

  <tag tag="input" type="button">
    <attr name="name">
      <content>Click "Me"!</content>
    </attr>
    <attr name="onclick">
      <content>alert("Thanks for testing quote behavior!")</content>
    </attr>
  </tag>

  <class name="discard-me">
    <attr name="class">
      <content>[[@/ discard-me//]]</content>
    </attr>
  </class>
</dynamator>
