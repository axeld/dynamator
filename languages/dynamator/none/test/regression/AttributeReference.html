<html>
<head>
  <title>Attribute References</title>
</head>
<body bgcolor="#ffffff">
<a href="x.htm">should be x.htm</a>
<br>
<a href="x.html">should be x.htm</a>
<br>
<a href="x.html#x">should be x.htm#x</a>
<br>
<img alt="junk">
<br>
<img>
<form>
<input type="text" name="test" title='Testing "quotes"'>
<input type="button" name="Click Me!" onclick="alert('Thanks!')">
</form>
<p class="keep-me discard-me">
Text.
</body>
</html>
