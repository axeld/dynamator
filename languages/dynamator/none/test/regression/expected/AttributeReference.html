<html>
  <head>
    <title>Attribute References</title>
  </head>
  <body bgcolor="#cccccc">
    <a href="x.htm">should be x.htm</a> <br>
    <a href="x.htm">should be x.htm</a> <br>
    <a href="x.htm#x">should be x.htm#x</a> <br>
    <img alt="This image shows: junk"> <br>
    <img alt="This image shows: "> 
    <form><input type="text" name="test" title='Testing "quotes"'> <input type="button" name='Click "Me"!' onclick='alert("Thanks for testing quote behavior!")'></form>
    <p class="keep-me">Text.</p>
  </body>
</html>
<!-- generated by Dynamator Sun Feb 15 17:23:23 CST 2004 -->
